libr/bin/dwarf.c

```diff
@@ -786,7 +786,7 @@ static const ut8 *parse_spec_opcode(
 		// line line-range information. move away
 		return NULL;
 	}
-	advance_adr = adj_opcode / hdr->line_range;
+	advance_adr = (adj_opcode / hdr->line_range) * hdr->min_inst_len;
```

Comment:
just to understand.. where does this `* min_inst_len` come from?

Reply:
From the dwarf standard, http://www.dwarfstd.org/doc/Dwarf3.pdf#page=111&zoom=100,0,0